app/src/forms/form/service.js

```diff
@@ -237,7 +237,7 @@ const service = {
       .withGraphFetched('idpHints')
       .throwIfNotFound()
       .then((form) => {
-        form.idpHints = form.idpHints.map((idp) => idp.code);
+        form.idpHints = form.idpHints.map((idp) => idp.idp);
```

Comment:
If I'm reading things correctly, this `idpHints` graph is from the `form_identity_provider` table, So this line 240 change is causing null to be returned for the IdPs. Is it more consistent in the long run to revert the line 240 change, or to change the name of the `form_identity_provider` column?

Reply:
I put this in the description and it is also in the documents, but not sure if I put a backlog ticket in? If there isn't one there should be.

> NOTE: when this is all stable, I think we should rename the db field and object from idp to hint and likewise update the userInfo object to use code and hint from idp and idpHint.

I did not want to do all that work in this PR. This is one area where it was very clear that `idp` and `code` were being used interchangeably and only worked because the values were the same.

Reply:
sigh, ok, I see what you are talking about (was right in the middle of something that used this). not sure why this worked and now doesn't. `code` certainly isn't correct (if the idp != code)...  not sure what exactly happened but yes, this needs fixing! I'll make one and push it up soon.

Reply:
No idea how this worked, best guess is that it actually didn't work and I just hit a case where it no longer failed like before but worked with a `null` value.  Anyway, fixing it!

Reply:
```
CountryMac:~ jason$ curl https://chefs-dev.apps.silver.devops.gov.bc.ca/pr-1258/api/v1/forms/2c7c02bd-9040-4315-bae7-04163424e2d3/options; echo
{"id":"2c7c02bd-9040-4315-bae7-04163424e2d3","name":"Auth - Public","description":"","idpHints":["public"],"snake":"auth_-_public"}
```

Reply:
`"idpHints":["public"]` is returned.